### ISSUE TITLE
Register names mixed with command register codes

### ISSUE THREAD

**Issue by gsmecher** [quartiq/xilinx-bitstream#1]
This code is not active, so it probably doesn't matter -- but I'm dropping an issue here just so you're aware.
The register definitions here:

```
registers = list(
    "CRC FAR FDRI FDRO CMD CTL0 MASK STAT LOUT COR0 MFWR CBC IDCODE "
    "AXSS COR1 _ WBSTAR TIMER BSPI_READ FALL_EDGE _ _ BOOTSTS _ CTL1 "
    "_ _ _ _ _ _ BSPI".split())
```

...are actually a mixture of registers (UG470 Table 5-23) and command register codes (UG470 Table 5-25). For example, BSPI_READ is a command that may be written to the CMD register, whereas CMD is the register itself. They're different animals and don't logically belong in the same list. (There are overlaps, too: FAR-the-register is address 1, which overlaps with command WCFG.)

At most, I think a warning comment in the code would be sufficient. Thanks for making the code public!

cheers,
Graeme

**Comment by jordens** [comment]
Sounds plausible. The only use case ever, that I'm aware of, was `Squeeze`, i.e. `IDCODE`.

Happy to accept PRs for this.

**Comment by gsmecher** [comment]
Fixed in #2

**Comment by gsmecher** [comment]
Thanks!
